Prace_domowe/Praca_domowa2/Piotr Marciniak/notebook.ipynb

```diff
@@ -0,0 +1,2120 @@
+{
```

Comment:
<p>Ciekawe obliczenia. Możliwe też by tu było zastosowanie macierzy rzadkich aby zaoszczędzić trochę pamięci, jednak to rozwiązanie też nie było zbyt dobre -- nie chcemy niepotrzebnie zwiększać wymiaru danych. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>